.editorconfig

```diff
@@ -0,0 +1,14 @@
+# Editor configuration normalization
+# @see http://editorconfig.org/
+
+# This is the top-most .editorconfig file; do not search in parent directories.
+root = true
+
+# All files.
+[*]
+end_of_line = LF
+indent_style = space
+indent_size = 4
```

Comment:
I am not sure whether these options are correct for this project. Is the indent style really set to 4 for all the files in the project? I see that the `.drone.yml` file for example is using an indent of 2 spaces. This is in conflict with what is being proclaimed here.

I would suggest to take this out of this PR for now, it is out of scope and seems to need additional work.